stable/datadog/values.yaml

```diff
@@ -1226,6 +1226,41 @@ kube-state-metrics:
     #
     create: true
 
+  ## @param image.tag - object - optional
+  image:
+    tag: v1.8.0
+
+  ## @param collectors - list of objects - optional
+  collectors:
+    certificatesigningrequests: true
+    configmaps: true
+    cronjobs: true
+    daemonsets: true
+    deployments: true
+    endpoints: true
+    horizontalpodautoscalers: true
+    ingresses: true
+    jobs: true
+    limitranges: true
+    mutatingwebhookconfigurations: true
+    namespaces: true
+    networkpolicies: true
+    nodes: true
+    persistentvolumeclaims: true
+    persistentvolumes: true
+    poddisruptionbudgets: true
+    pods: true
+    replicasets: true
+    replicationcontrollers: true
+    resourcequotas: true
+    secrets: true
+    services: true
+    statefulsets: true
+    storageclasses: true
+    validatingwebhookconfigurations: true
+    verticalpodautoscalers: false
+    volumeattachments: true
+
```

Comment:
Like for the `README.md`, I think it would be simpler to not override the default values that are coming from the KSM upstream chart.

Reply:
The reason behind the override is because some features are not available in certain clusters, forcing users having to wait for a new release. Meanwhile, we're getting spam for things like verticalpodautoscalers, mutatingwebhookconfigurations, etc...